src/rules/style.ts

```diff
@@ -13,6 +13,8 @@ export default {
   'arrow-parens': true,
   // Suggests to convert () => { return x; } to () => x.
   'arrow-return-shorthand': true,
+  // In a binary expression, a literal should always be on the right-hand side if possible. For example, prefer ‘x + 1’ over ‘1 + x’.
+  'binary-expression-operand-order': true,
```

Comment:
👍 

I guess this is equivalent to [eslint's yoda](http://eslint.org/docs/rules/yoda)?  So having this on is in line with our eslint rules.